backend/package.go

```diff
@@ -114,7 +114,7 @@ func asBool(s string) bool {
 func str2map(s string) map[string]string {
 	ret := map[string]string{}
 
-	for _, kv := range strings.Split(s, " ") {
+	for _, kv := range strings.Split(s, ",") {
```

Comment:
Can this instead be a regexp split on `[ ,]`?  The `TMPFS_MAP` config option is a case that's already being used by some Travis Enterprise installations.  If we want to be really thorough, I think that each item in `kvParts` should also be URI-decoded :grin:

Reply:
@meatballhat :+1: see latest commits